lib/src/widgets/country_selector/country_selector_navigator.dart

```diff
@@ -428,6 +428,11 @@ class DraggableModalBottomSheetNavigator extends CountrySelectorNavigator {
         maxChildSize: maxChildSize,
         expand: false,
         builder: (context, scrollController) {
+          // hide the keyboard when scrolling the list, so it doesn't overlap the list
+          scrollController.addListener(() {
```

Comment:
Is the scroll controller disposed somewhere, it does not seem to be in this PR ? 

  1. It's a bit odd to have the listener added here and removed somewhere else
  2. I believe this will add a listener on every rebuild

Reply:
@simplenotezy do you plan on resolving this in the short term ? Asking for a future release

Reply:
@cedvdb excuse me for the latency 😅 however, now we're removing the listener on dispose. See latest commit. This was the best way I could think of for now (creating a new widget, so we can get access to the lifecycle methods)

Reply:
I have also aligned the PR with dev branch so I think it's safe to resolve this conversation now

Reply:
@cedvdb by the way, listeners are automatically cleared on dispose:

![image](https://github.com/cedvdb/phone_form_field/assets/3786627/ebf00f64-b661-4387-9584-3b7bc06a5671)